src/NewCommand.php

```diff
@@ -243,4 +283,73 @@ protected function findComposer()
 
         return 'composer';
     }
+
+    /**
+     * @param  string  $packages
+     * @param  string  $composer
+     * @return  string
+     */
+    private function getLaravelPackages($packages, $composer)
+    {
+        $laravelPackages = explode(',', $packages);
+
+        $composerPackages = '';
+        foreach ($laravelPackages as $laravelPackage) {
+            if (array_key_exists($laravelPackage, self::LARAVEL_PACKAGES)) {
+                $composerPackages .= ' '.self::LARAVEL_PACKAGES[$laravelPackage];
+            }
+        }
+
+        if ($composerPackages !== '') {
+            return $composer.' require'.$composerPackages;
+        }
+
+        return $composerPackages;
+    }
+
+    /**
+     * @param  string  $packages
+     * @param  string  $composer
+     * @return  string
+     */
+    private function getLaravelDevPackages($packages, $composer)
+    {
+        $laravelPackages = explode(',', $packages);
+
+        $composerPackages = '';
+        foreach ($laravelPackages as $laravelPackage) {
+            if (array_key_exists($laravelPackage, self::LARAVEL_DEV_PACKAGES)) {
+                $composerPackages .= ' '.self::LARAVEL_DEV_PACKAGES[$laravelPackage];
+            }
+        }
+
+        if ($composerPackages !== '') {
+            return $composer.' require --dev'.$composerPackages;
+        }
+
+        return $composerPackages;
+    }
+
+    /**
+     * @param  string  $packages
+     * @return  string
+     */
+    private function getLaravelInstallCommands($packages)
+    {
+        $laravelPackages = explode(',', $packages);
+
+        $packagesInstallCommands = '';
+
+        foreach ($laravelPackages as $laravelPackage) {
+            if (array_key_exists($laravelPackage, self::LARAVEL_PACKAGES_INSTALL_COMMAND)) {
```

Comment:
`array_key_exists` is deprecated: https://www.php.net/manual/es/migration74.deprecated.php#migration74.deprecated.core.array-key-exists-objects.

Reply:
Do you recommend use `isset`?

Reply:
Yes.